lib/matplotlib/mathtext.py

```diff
@@ -768,22 +768,17 @@ class UnicodeFonts(TruetypeFonts):
     use_cmex = True
 
     def __init__(self, *args, **kwargs):
-        # This must come first so the backend's owner is set correctly
```

Comment:
I think the comment is still valid. May change to

~~~
 ... must come before __init__ ...
~~~